hbase-shaded/pom.xml

```diff
@@ -441,8 +441,8 @@
 
                                     <!-- top level net-->
                                     <relocation>
-                                        <pattern>net</pattern>
-                                        <shadedPattern>${shaded.prefix}.net</shadedPattern>
+                                        <pattern>net.</pattern>
```

Comment:
I think this should be "net/" instead of "net." and then the shade plugin will look for both because it takes the slash to mean  like a package delimiter.